### PR TITLE
Feature: Inbox navigation

### PR DESCRIPTION
**Pull request by maxplt**
# Jira Issue: [IP-286](https://hedvig.atlassian.net/jira/software/projects/IP/boards/19?selectedIssue=IP-286)

## What?
Implement keyboard navigation to the **inbox page**.

Checklist:
- [x] First listing item
- [x] `Arrow key up/down` -> nav along the list & change chatroom accordingly
- [x] `Enter` -> focus on text field
- [x] `Esc` -> shift focus back to the list
- [x] `Option + M `-> open member profile

## Why?
For keyboard coverage

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: src/portals/hope/features/tasks/list-items/ClaimTaskListItem.tsx [hunk 1]
```diff
@@ -44,7 +45,7 @@ const MemberFlag = styled.div`
 `
 
 export const ClaimTaskListItem: React.FC<
-  TaskListItemExtension<ClaimTaskResourceFragment>
+  TaskListItemExtension<ClaimTaskResourceFragment> & HTMLMotionProps<'li'>
 > = ({ task, ...props }) => {
```
Comment (falklinus) [inline src/portals/hope/features/tasks/list-items/ClaimTaskListItem.tsx:49]: Sweet

File: src/portals/hope/features/tasks/list-items/ClaimTaskListItem.tsx [hunk 2]
```diff
@@ -44,7 +45,7 @@ const MemberFlag = styled.div`
 `
 
 export const ClaimTaskListItem: React.FC<
-  TaskListItemExtension<ClaimTaskResourceFragment>
+  TaskListItemExtension<ClaimTaskResourceFragment> & HTMLMotionProps<'li'>
```
Comment (falklinus) [inline src/portals/hope/features/tasks/list-items/ClaimTaskListItem.tsx:48]: Sweet

**Review by falklinus — APPROVED** [review]

**Comment by falklinus** [comment]
Nice update, I like the motion types 🔥

File: src/portals/hope/pages/tasks/TasksPage.tsx [hunk 3]
```diff
@@ -314,6 +314,36 @@ const TasksPage: Page = () => {
...
+    }),
+    name: 'Inbox',
+    nameField: 'id',
+    autoFocus: true,
+    resolve: 'TasksChatInput',
+    onNavigation: (item) => {
+      navigate({ taskId: item.id })
+    },
+    styles: {
+      focus: { border: '2px solid red' },
```
Comment (vonElfvin) [inline src/portals/hope/pages/tasks/TasksPage.tsx:333]: `solid red` sounds a bit ugly 😄

**Review by vonElfvin — COMMENTED** [review]
Nice! Added some smaller comments :)

Any gif?

**Pull request closed without merging**